hazelcast/src/main/java/com/hazelcast/nio/tcp/handlermigration/IOBalancer.java

```diff
@@ -178,26 +179,4 @@ private void tryMigrate(LoadImbalance loadImbalance) {
         }
         handler.migrate(destinationSelector);
     }
-
-    private final class IOBalancerThread extends Thread {
```

Comment:
Why did you move the thread to the outside? It does almost nothing and it relies on the internals of the balancer.